usagov/usagov-2021
# USAGOV-1800-select-all-spacing

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1800

## Description

Fix spacing around select all checkbox when error alert is displayed on mobile.
Fix button sizing breakpoints

## Type of Changes

![image](https://github.com/user-attachments/assets/1ee472a9-445c-4505-88f8-d7bd32814605)

![image](https://github.com/user-attachments/assets/afc0dbe2-6a2e-4d69-9c49-bef171aee9d8)

![image](https://github.com/user-attachments/assets/c4dc7ee7-dfdb-4365-bb1d-8bc395381102)


## Testing Instructions


Need benefit search page set up and published

### Change Requirements



### Validation Steps

- On mobile width, when page loads space below "Select all" is same as space between other checkboxes. Button is above the other category checkboxes.
- On mobile width, when error is shown, space below "Select all" is same as space between other checkboxes. 
- Desktop layout is unchanged, "Select all" button should be in right-most column by itself.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
